### PR TITLE
Test merge bot

### PR DESCRIPTION
**Pull request by Mothblocks**
test merge bot test merge bot test merge bot test merge bot

### CODE REVIEW

File: config/config.txt [hunk 1]
```diff
@@ -609,7 +609,7 @@ MOTD motd.txt
 ## The cache is assumed to be cleared by TGS recompiling, which deletes `tmp`.
 ## This should be disabled (through `CACHE_ASSETS 0`) on development,
 ## but enabled on production (the default).
-CACHE_ASSETS 0
+#CACHE_ASSETS 0
```
Comment (LemonInTheDark) [inline config/config.txt:612]: ?????

File: config/game_options.txt [hunk 2]
```diff
@@ -453,13 +453,13 @@ BOMBCAP 20
...
 ## of ruins.
-LAVALAND_BUDGET 60
+LAVALAND_BUDGET 0
 
 ## Ice Moon Budget
 ICEMOON_BUDGET 90
 
 ## Space Ruin Budget
-Space_Budget 16
+Space_Budget 0
```
Comment (LemonInTheDark) [inline config/game_options.txt:463]: mothhhhhh

File: .prettierignore [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # We don't want prettier to run on anything outside of the TGUI folder, so we have to do this.
-/*
+# /*
```
Comment (LemonInTheDark) [inline .prettierignore:2]: ?????????

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

**Comment by Mothblocks** [comment]
I meant to commit this to my own repo I'm sorry for spoiling the surprise

File: tools/test_merge_bot/config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// A URL that, on GET, will return an array of objects with the schema
+// { round_id, datetime, test_merges, server, url }
+// MOTHBLOCKS TODO: Link the mothbus code for this as an example
+export const GET_TEST_MERGES_URL =
+	// "https://bus.moth.fans/recent-test-merges.json";
+	"http://6ffc-172-92-14-65.ngrok.io/recent-test-merges.json";
```
Comment (optimumtact) [inline tools/test_merge_bot/config.js:6]: nodders

Comment (Mothblocks) [reply tools/test_merge_bot/config.js:6]: ive gotten very very good at making these fast

**Pull request closed without merging**